### ISSUE TITLE
Make OpenBrowser optional when starting dev environment

### ISSUE THREAD

**Issue by lightbringer1991** [Adslot/adslot-ui#613]
In Linux server version, there is no graphic library and therefore browsers cannot be installed.
`npm start` will throw error when it tries to open the default browser. Error details below:

```bash
Error: Command failed: /var/www/nodejs/adslot-ui/node_modules/open/vendor/xdg-open "http://localhost:8888/webpack-dev-server/"
xdg-open: no method available for opening 'http://localhost:8888/webpack-dev-server/'

    at ChildProcess.exithandler (child_process.js:198:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:920:16)
    at Socket.<anonymous> (internal/child_process.js:351:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:497:12)
```

Proposal: add an option to NOT open the browser if needed, so by default it will still do what it does now
e.g. `OPEN_BROWSER=false npm start`

**Comment by vinteo** [comment]
you can use `npm run start:cold`

**Comment by lightbringer1991** [comment]
@vinteo I think `start:cold` is broken atm, since the actual command is `"start:cold": "node server.js --env=dev-cold"`, and there is no `server.js`

I think it should be changed to 
`"start:cold": "webpack-dev-server --config=./config/dev-cold.js"`
(with some changes in `dev-cold.js`, since it's not working atm)

Also it doesn't provide hot reload, which I guess why it's called `cold`. However I would like to still have hot reload.

This is what I setup for devevelopment:
- Ubuntu 16.04 server in VMWare (no GUI)
- Webstorm in Windows host, editing the code via SMB/samba

so basically all code editing/troubleshooting is from Windows, and interact with the node server remotely(ish). That's why I'd like hot reload as well, just don't open a browser when starting dev env.

However I think this can be fixed via the deprecation of `OpenBrowserPlugin`, as stated in https://github.com/Adslot/adslot-ui/issues/614
